package.json

```diff
@@ -42,7 +42,7 @@
     "offline-plugin": "4.8.3",
     "prop-types": "15.5.10",
     "react": "15.6.1",
-    "react-apollo": "1.4.10",
+    "react-apollo": "^1.4.11",
```

Comment:
On RAN!, Plugins need to run on a specific version. Not with the caret. Otherwise, Plugins can be a different version on each developer env. So Please, remove caret chars from the package versions.